apps/src/lib/levelbuilder/data-docs-editor/DataDocFormEditor.jsx

```diff
@@ -0,0 +1,93 @@
+import React, {useState} from 'react';
+import PropTypes from 'prop-types';
+import RailsAuthenticityToken from '@cdo/apps/lib/util/RailsAuthenticityToken';
+import HelpTip from '@cdo/apps/lib/ui/HelpTip';
+import TextareaWithMarkdownPreview from '@cdo/apps/lib/levelbuilder/TextareaWithMarkdownPreview';
+import {navigateToHref} from '@cdo/apps/utils';
+
+const DataDocFormEditor = props => {
+  const {dataDocKey, originalDataDocName, originalDataDocContent} = props;
+  const [dataDocContent, setDataDocContent] = useState(originalDataDocContent);
+  const [dataDocName, setDataDocName] = useState(originalDataDocName);
+
+  const save = () => {
+    $.ajax({
+      url: `/data_docs/${dataDocKey}`,
+      method: 'PUT',
+      data: {
+        name: dataDocName,
+        content: dataDocContent
+      }
+    }).done(() => {
+      navigateToHref(`/data_docs/${dataDocKey}`);
+    });
+  };
+
+  return (
+    <div>
+      <RailsAuthenticityToken />
+      <h1>Edit Data Doc</h1>
+      <h2>
+        This feature is in progress. It is not ready for use on Levelbuilder
```

Comment:
We forgot to remove this for a while for Code Docs so it would be great to make sure we have a work item tracked to remove this when we launch

Reply:
Added here: https://codedotorg.atlassian.net/browse/PLAT-1998